hudi-client/src/main/java/org/apache/hudi/execution/BulkInsertMapFunction.java

```diff
@@ -51,6 +51,7 @@ public BulkInsertMapFunction(String instantTime, HoodieWriteConfig config, Hoodi
   @Override
   public Iterator<List<WriteStatus>> call(Integer partition, Iterator<HoodieRecord<T>> sortedRecordItr) {
     return new CopyOnWriteLazyInsertIterable<>(sortedRecordItr, config, instantTime, hoodieTable,
-        fileIDPrefixes.get(partition));
+        fileIDPrefixes.get(partition), hoodieTable.getIdSupplier(), hoodieTable.getStageSupplier(),
```

Comment:
`hoodieTable.getIdSupplier()` is not clear here. I suggest we can rename these getter to e.g. `getPartitionIdSupplier`, `getStageId` and `getAttemptId`?